assets/js/listener.js

```diff
@@ -17,17 +21,48 @@ document.addEventListener('click', function(event) {
     }
 });
 
+document.getElementById("comment").addEventListener("input", function() {
+    const text = this.value.trim();
+    const sendButton = document.getElementById('send');
+    
+    if (!document.getElementById('yes').checked && !document.getElementById('no').checked) {
```

Comment:
It is wasteful to ask the document to go find these elements with each keystroke. We should store these references to elements in variables outside the function and use those variables instead.

We should do the same to all other functions too.

Reply:
Thank you! Refactored.